### ISSUE TITLE
Lost the ability to click URL and lost Parentheses matching

### ISSUE THREAD

**Issue by icuurd12b42** [Microsoft/vscode#34926]
- VSCode Version: Code 1.16.1 (27492b6bf3acb0775d82d2f87b25a93490673c6d, 2017-09-14T16:38:23.027Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-autohotkey|sle|0.2.1

I Lost the ability to click URLs in code and lost Parentheses matching feature. not sure if I hit a hot key or something... both of these stopped working

And how can I disable ctlr+b it's really annoying on missed hit ctrl+v
and ctrl+c copying the entire line when nothing is selected I would like to disable that too

---

Steps to Reproduce:

1.
2.

Reproduces without extensions: Yes/No

**Comment by yume-chan** [comment]
I cannot reproduce your problems on 1.16.1 and 1.17-insiders. Please make sure your `editor.links` and `editor.matchBrackets` settings are all `true`.

To disable <kbd>Ctrl</kbd> + <kbd>B</kbd>, change your keybindings (press <kbd>Ctrl</kbd> + <kbd>K</kbd> <kbd>Ctrl</kbd> + <kbd>S</kbd> to open keybindings, search for <kbd>Ctrl</kbd> + <kbd>B</kbd>, click on the item then press <kbd>Delete</kbd>).

To disable copy empty selection, change your settings (press <kbd>Ctrl</kbd> + <kbd>,</kbd> to open settings, search for and hover on `editor.emptySelectionClipboard`, click the little pen icon on the left and then select `false`)

**Comment by icuurd12b42** [comment]
Alright thanks. Fixed....

The click link and the match brackets came back when I restarted my PC

**Comment by alexandrudima** [comment]
@icuurd12b42 Perhaps you had a Screen Reader active? [We show a green message in the status bar]. We turn off certain features that do not interact well with Screen Readers. You can disable this behaviour by using `"editor.accessibilitySupport": "off"`.

**Comment by icuurd12b42** [comment]
YES. I'm coding a project that uses MS SAPI and that green message did pop up when I launched MS Speech Recognition to change the mic settings and do some training! I can't remember how I dismissed that green message. I might have been curious to see how it worked in code and totally forgot to give it a shot, more concern about my own stuff...

You should look at your code to detect when the reader goes away :)

**Comment by alexandrudima** [comment]
@icuurd12b42 That's a limitation on Windows, there's no way to detect when a screen reader goes away... In fact even detecting that a screen reader is attached is ... a bit ... magic...

**Comment by alexandrudima** [comment]
Added a dialog to clarify things in #32166

**Comment by icuurd12b42** [comment]
>detecting that a screen reader is attached is ... a bit ... magic...

I can imagine.  I had to do sorcery myself which sortof interfered with your spell
